packages/buefy-next/src/components/colorpicker/Colorpicker.vue

```diff
@@ -160,6 +162,10 @@ export default {
                     )
             }
         },
+        mobileNative: {
```

Comment:
This new prop looks unrelated to the solution. Please remove these lines.

Reply:
If you would please attempt to remove that prop. What you will find is that you will get an error regarding that prop.

Reply:
Sorry, I did not notice you are fixing both #34 and #35 in this PR.

Reply:
> Sorry, I did not notice you intend to fix both #34 and #35 in this PR.

Yeah that's was my fault because I forgot to update the description when I updated the title🫣